codalab_service.py

```diff
@@ -215,7 +215,8 @@ def has_callable_default(self):
         default=100,
     ),
     ### Email
-    CodalabArg(name='admin_email', help='Email to send admin notifications to (e.g., monitoring)'),
+    CodalabArg(name='monitor_email', help='Monitoring email to send notifications to'),
+    CodalabArg(name='help_email', help='Help email to send user questions to'),
```

Comment:
Maybe call this `admin_email` and `support_email` to use consistent terminology with what's used in the `config.json` file.